src/renderer/reader/components/ReaderMenu.tsx

```diff
@@ -613,6 +615,9 @@ const AnnotationList: React.FC<{ r2Publication: R2Publication, dockedMode: boole
         }
     }, [isSearchEnable]);
 
+    if (!r2Publication || !annotationsQueue) {
```

Comment:
`!annotationsQueue?.length`, I think

Reply:
I don't think so either

Reply:
removed

Reply:
falsy `annotationsQueue` from IReaderRootState[reader.annotation] results in empty JSX, but empty array too, no?